packages/airnode-deployer/src/utils/validation.test.ts

```diff
@@ -1,17 +1,20 @@
+import { mockConsole } from '../../test/mock-utils';
 import fs from 'fs';
 import { loadConfig } from './files';
 import * as fixtures from '../../test/fixtures';
 
+mockConsole();
+
 describe('deployer-validation', () => {
-  it('loads the config without validation', () => {
+  it('loads the config without validation', async () => {
```

Comment:
`console.log `seems to only  work here if this is `async` which seems weird

Reply:
That is very wierd.